File: MTMA.API/MTMA.API/MTMA.Services.ServiceModels/Services/Identity/RegisterUserServiceModel.cs
namespace MTMA.Services.ServiceModels
{
    using MTMA.Data.Models.Identity;
    using MTMA.Services.Mapping;

    public class RegisterUserServiceModel : IMapTo<MTMAUser>
    {
        public string Username { get; set; } = string.Empty;

        public string Email { get; set; } = string.Empty;

        public string Password { get; set; } = string.Empty;

        public string ConfirmPassword { get; set; } = string.Empty;
    }
}

File: MTMA.UI/mock_tests/lib/core/service_models/identity/register_user_service_model.dart
class RegisterServiceModel {
  String username;
  String email;
  String password;
  String confirmPassword;

  RegisterServiceModel({
    required this.username,
    required this.email,
    required this.password,
    required this.confirmPassword,
  });

  Map<String, String> get toJson => {
        "Username": username,
        "Email": email,
        "Password": password,
        "ConfirmPassword": confirmPassword,
      };
}
